packages/amplify_api/android/build.gradle

```
@@ -0,0 +1,45 @@
+group 'com.amazonaws.amplify.amplify_api'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.3.50'
```

Should you use the latest version here?  
```suggestion
    ext.kotlin_version = '1.4.21'
```